core/src/main/scala/com/yahoo/maha/core/BaseUTCTimeProvider.scala

```diff
@@ -286,9 +289,18 @@ class BaseUTCTimeProvider extends UTCTimeProvider with Logging {
 
   def validateFilters(dayFilter: Filter, hourFilter: Option[Filter], minuteFilter: Option[Filter]): Boolean = {
     if (!hourFilter.isDefined && !minuteFilter.isDefined) return true
-    if (!hourFilter.isDefined && minuteFilter.isDefined) return false
-    if (hourFilter.isDefined && dayFilter.operator != hourFilter.get.operator) return false
-    if (hourFilter.isDefined && minuteFilter.isDefined && hourFilter.get.operator != minuteFilter.get.operator) return false
+    if (!hourFilter.isDefined && minuteFilter.isDefined) {
```

Comment:
all of these logs can be a single log line with condition checking inside it.

Reply:
As discussed offline, not fixing this.